apps/asap-server/src/controllers/calendars.ts

```diff
@@ -91,6 +102,51 @@ export default class Calendars implements CalendarController {
     return res.items[0];
   }
 
+  fetchById(id: string, options?: { raw: false }): Promise<CalendarResponse>;
```

Comment:
you might wonder why i added this method - well at first i thought i'd use it in the calendar-handler to fetch the newest calendar data (atm we get it from the event payload). Turns out we need to stick to what we have because of how google do subscriptions - we have to be able to compare the old and new payload each time to cancel subscriptions etc.

I figured since I wrote the method I might as well leave it here